app/celery/tasks.py

```diff
@@ -214,6 +214,7 @@ def process_rows(rows: List, template: Template, job: Job, service: Service):
                 "template_version": job.template_version,
                 "job": str(job.id),
                 "to": row.recipient,
+                "reply_to_text": dao_get_reply_to_by_id(service.id, sender_id).email_address,
```

Comment:
running locally I'm getting a failure if the service does not have a custom reply_to set (where I think `sender_id` will be `None`)
```
 File "/workspace/app/celery/tasks.py", line 132, in process_job
    process_rows(result, template, job, service)
  File "/workspace/app/celery/tasks.py", line 217, in process_rows
    "reply_to_text": dao_get_reply_to_by_id(service.id, sender_id).email_address,
  File "/workspace/app/dao/service_email_reply_to_dao.py", line 25, in dao_get_reply_to_by_id
    db.session.query(ServiceEmailReplyTo)
  File "/home/vscode/.local/lib/python3.9/site-packages/sqlalchemy/orm/query.py", line 3500, in one
    raise orm_exc.NoResultFound("No row was found for one()")
sqlalchemy.orm.exc.NoResultFound: No row was found for one()
```

Reply:
I have looked into this and possibly need a separate test for this scenario